source/refills/tooltipped.html.erb

```diff
@@ -0,0 +1,7 @@
+<a href="#0" class="tooltipped" aria-label="View Notifications">
+  <svg aria-hidden="true" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
@creuter I think I was getting tripped up by the last little bit in that description. Thoughts on being a bit more verbose?

```
…like links and buttons which are only represented graphically.
```